### ISSUE TITLE
Song details becomes '?' when the song name is not English

### ISSUE THREAD

**Issue by seromy** [AngryKiller/iTunes-bridge#9]
Hi, first of all, thank you for making this such a good package for us :)

Problem: Not able to show non-ASCII characters

Details: The song name in the attached image is originally call '早送りカレンダー ' in Japanese, but the console output ???????? and my application also show this.
![Screenshot 2020-11-04 183818](https://user-images.githubusercontent.com/43441144/98102089-1cc99000-1ece-11eb-982e-5fac7561adc7.jpg)
⬆Console capture

![Screenshot 2020-11-04 202631](https://user-images.githubusercontent.com/43441144/98111772-18a46f00-1edc-11eb-97e3-78079be51fe5.jpg)
⬆The song info box in iTunes

![Screenshot 2020-11-04 202843](https://user-images.githubusercontent.com/43441144/98112180-b435df80-1edc-11eb-863e-b9f6b73a73dd.jpg)
⬆Sample of using currentTrack.name inside an electron app.

How can I solve this problem or this is a bug?
<hr/>
Electron version: v8.3.0<br>
Nodejs version: v12.18.3<br>
npm version: 6.14.6

**Comment by AngryKiller** [comment]
Hello and sorry for the long delay.
I'm gonna see if this problem also happens on macOS, if it isn't the case then it's probably an issue with WScript on Windows and I'm gonna try to find a workaround.

**Comment by 7loud** [comment]
```js
json = {
            "name": encodeURIComponent(currentTrack.name),
            "artist": encodeURIComponent(currentTrack.artist),
            "album": encodeURIComponent(currentTrack.album),
            "mediaKind": currentTrack.kind,
            "duration": currentTrack.duration,
            "elapsedTime": iTunesApp.PlayerPosition,
            "remainingTime": remainingTime,
            "genre": encodeURIComponent(currentTrack.genre),
            "releaseYear": currentTrack.year,
            "id": encodeURIComponent(currentTrack.name), // I haven't found a way to get the current track ID with iTunes COM :/
            "playerState": playerState
        };
```
```js
decodeURIComponent(currentTrack.name)
decodeURIComponent(currentTrack.artist)
decodeURIComponent(currentTrack.album)
...
```

Not the best way, but it works
![image](https://user-images.githubusercontent.com/53608074/120075243-bcd04c00-c0a0-11eb-8c29-029aba92bc1b.png)

**Comment by AngryKiller** [comment]
> ```js
> ```js
> json = {
>             "name": encodeURIComponent(currentTrack.name),
>             "artist": encodeURIComponent(currentTrack.artist),
>             "album": encodeURIComponent(currentTrack.album),
>             "mediaKind": currentTrack.kind,
>             "duration": currentTrack.duration,
>             "elapsedTime": iTunesApp.PlayerPosition,
>             "remainingTime": remainingTime,
>             "genre": encodeURIComponent(currentTrack.genre),
>             "releaseYear": currentTrack.year,
>             "id": encodeURIComponent(currentTrack.name), // I haven't found a way to get the current track ID with iTunes COM :/
>             "playerState": playerState
>         };
> ```
> 
> 
>     
>       
>     
> 
>       
>     
> 
>     
>   
> ```js
> decodeURIComponent(currentTrack.name)
> decodeURIComponent(currentTrack.artist)
> decodeURIComponent(currentTrack.album)
> ...
> ```
> 
> 
>     
>       
>     
> 
>       
>     
> 
>     
>   
> Not the best way, but it works
> ```

Thanks for that fix! You can create a pull request if you want and I will implement it.

**Comment by seromy** [comment]
Thank you so much!

**Comment by AngryKiller** [comment]
@itztaylorau can you confirm that the issue is resolved using iTunes-bridge 0.6.7?

**Comment by seromy** [comment]
> @itztaylorau can you confirm that the issue is resolved using iTunes-bridge 0.6.7?

Yes and I can confirm that the latest version 0.6.7 works with special characters. <br>
Sample:
![Screenshot 0003-06-19 at 02 59 22](https://user-images.githubusercontent.com/43441144/122605914-af5c1f80-d0aa-11eb-8006-13f5559dacf3.png)
